src/NHibernate/Logging.cs

```diff
@@ -146,15 +162,15 @@ private static INHibernateLoggerFactory GetLoggerFactory(string nhibernateLogger
 			try
 			{
 				var loadedLoggerFactory = Activator.CreateInstance(loggerFactoryType);
-#pragma warning disable 618
-				if (loadedLoggerFactory is ILoggerFactory oldStyleFactory)
+				if (loadedLoggerFactory is INHibernateLoggerFactory newStyleFactory)
```

Comment:
Test inverted because log4net logger factory does now implement both interfaces (old and new).